### PR TITLE
added named query context and request log level error when error.

### PR DESCRIPTION
**Pull request by saito-sv**

### CODE REVIEW

**Review by ocasta181 — APPROVED** [review]

File: middleware/middleware.go [hunk 1]
```diff
@@ -55,7 +55,14 @@ func LogRequest() echo.MiddlewareFunc {
 			if !ok {
 				logger.Warn().Msg("no span found in context")
 			}
-			logger.Info().
+			var logEnvent *zerolog.Event
```
Comment (Auroter) [inline middleware/middleware.go:58]:
```suggestion
			var logEvent *zerolog.Event
```

File: middleware/middleware.go [hunk 2]
```diff
@@ -55,7 +55,14 @@ func LogRequest() echo.MiddlewareFunc {
...
+			if v.Error != nil {
+				logEnvent = logger.Error().Err(v.Error)
```
Comment (Auroter) [inline middleware/middleware.go:60]:
```suggestion
				logEvent = logger.Error().Err(v.Error)
```

File: middleware/middleware.go [hunk 3]
```diff
@@ -55,7 +55,14 @@ func LogRequest() echo.MiddlewareFunc {
...
+			} else {
+				logEnvent = logger.Info()
```
Comment (Auroter) [inline middleware/middleware.go:62]:
```suggestion
				logEvent = logger.Info()
```

File: middleware/middleware.go [hunk 4]
```diff
@@ -55,7 +55,14 @@ func LogRequest() echo.MiddlewareFunc {
...
+			}
+
+			logEnvent.
```
Comment (Auroter) [inline middleware/middleware.go:65]:
```suggestion
			logEvent.
```

**Review by Auroter — CHANGES_REQUESTED** [review]
I think 'envent' is a misspelling of 'event', unless 'envent' is some kind of senior engineering term I haven't come across yet

**Review by Auroter — APPROVED** [review]

**Pull request merged**